assets/src/dashboard/components/alert/index.js

```diff
@@ -13,10 +13,49 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
this whole file used to be alert/container.js - no need for that file anymore or `Wrapper` so shrunk it up